Examples/Examples-iOS/IGListKitExamples/SectionControllers/SearchSectionController.swift

```diff
@@ -46,14 +46,13 @@ final class SearchSectionController: ListSectionController, UISearchBarDelegate,
     }
 
     func searchBarTextDidEndEditing(_ searchBar: UISearchBar) {
-        delegate?.searchSectionController(self, didChangeText: "")
+        delegate?.searchSectionController(self, didChangeText: searchBar.text!)
```

Comment:
Can you give a little detail about why we want to make this change?

Reply:
If the user starts scrolling, the `searchBar` will be resigned. This will trigger the `searchBarTextDidEndEditing` function. Now since we are calling the delegate function with hard coded empty string it will show all the items in the list.